deps/ops/public/Ops_Constants/OpsMod_ObsGroupInfo.f90

```diff
@@ -72,7 +72,8 @@ MODULE OpsMod_ObsGroupInfo
 INTEGER, PARAMETER :: ObsGroupOceanWinds   =  61 ! => Satellite wind speeds
 INTEGER, PARAMETER :: ObsGroupGIIRSLW      =  62 ! => GIIRS LW radiances
 INTEGER, PARAMETER :: ObsGroupGIIRSMW      =  63 ! => GIIRS MW radiances
-INTEGER, PARAMETER :: max_obs_group_num    =  63 ! Number of observation groups
+INTEGER, PARAMETER :: ObsGroupScatwindChosen =  64 ! => Scatterometer winds (single solution)
```

Comment:
Should we align all other `=` for obs with this one seeing as the name for this one is longer?